.gitlab-ci.yml

```diff
@@ -1074,15 +1095,10 @@ test_full_integration:
     - tar -xf /tmp/workspace.tar.gz
     - mv /tmp/build_revisions.env /tmp/stage-artifacts .
 
-    # Post job status
-    - apk --update add curl jq
-    - ${CI_PROJECT_DIR}/scripts/github_pull_request_status pending "Gitlab ${CI_JOB_NAME} started" "${CI_JOB_URL}" "${CI_JOB_NAME}/${INTEGRATION_REV}"
-
     # Output storage io stats
-    - apk add hdparm
     - df -h . | tail -1 | awk '{system("hdparm -tT "$1);}'
     # Get and install the integration test requirements
-    - apk add $(cat ${WORKSPACE}/integration/tests/requirements/apk-requirements.txt)
+    - apk add --no-cache $(cat ${WORKSPACE}/integration/tests/requirements/apk-requirements.txt)
```

Comment:
Any particular reason for this? I think we should not use `--no-cache`. See ea7223f078ab45b8b248c579e741b13c4affdc80.

This applies to all `--no-cache`s added.

Reply:
*Reverted*